js/twemoji-picker.js

```diff
@@ -1100,6 +1101,10 @@
             } else if (document.selection && document.selection.type != 'Control') {
                 document.selection.createRange().pasteHTML(text);
             }
+            
+            if(this.options.closePickerAfterSelection) {
+                this.closePicker();    
```

Comment:
I think the function `pasteAtCursor` should keep doing the paste only, so I would put this logic inside this event :

```
this.$twemojiList.find('img').on('click', function() {
    self.copyTwemoji($(this));
 });
```